### ISSUE TITLE
ioprio and renice options are not reset on unregister

### ISSUE THREAD

**Issue by mdiluz** [FeralInteractive/gamemode#141]
**Describe the bug**
In `game_mode_context_register` we set up these three optimisations:

```
	/* Apply scheduler policies */
	game_mode_apply_renice(self, client);
	game_mode_apply_scheduling(self, client);

	/* Apply io priorities */
	game_mode_apply_ioprio(self, client);
```

But there is no equivalent set of calls in `game_mode_context_unregister`. 

This will need a bit of nuance as the process may be changing these values itself, but all three should be able to be reset when appropriate. 

**To Reproduce**
Steps used to reproduce the behavior:
1. Set `ioprio` and `renice` in `gamemode.ini`
2. Request gamemode start from process
3. Request gamemode end from process
4. Note that I/O priority, niceness params are still set.

**Expected behavior**
Values should be re-set as appropriate.

**Comment by mdiluz** [comment]
I've moved the `softrealtime` issue out to #139

**Comment by mdiluz** [comment]
#142 has the fix for this

**Comment by aejsmith** [comment]
Merged #142.
